defaults/main.yml

```diff
@@ -621,6 +621,7 @@ postgresql_search_path: # schema names
   - "public"
 postgresql_default_tablespace: "" # a tablespace name, "" uses the default
 postgresql_temp_tablespaces: [] # a list of tablespace names
+postgresql_tablespaces_dirs: [] # a list of directories for tablespaces to be created
```

Comment:
It's a bit nitpicky @jasonrogena ... but should the variable be called `postgresql_tablespace_dirs`, rather than a double plural?